ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -455,11 +456,15 @@ private static Comparable getAttributeValue(Result result, String attributeName)
         queryTransform.getColumnAliasMap() != null
             ? queryTransform.getColumnAliasMap()
             : Collections.emptyMap();
-
+    
+    Serializable transformerId =
```

Comment:
✏️ Alternatively, and with 💯 % less `!=`: `Optional.ofNullable(arguments.get("transformerId")).orElse("")`